core/base/src/test/java/alluxio/collections/PairTest.java

```diff
@@ -11,7 +11,11 @@
 
 package alluxio.collections;
 
-import org.junit.Assert;
+import static
+org.junit.Assert.assertEquals;
```

Comment:
this should go on the same line as `import static` - see other uses of import static in this repository for reference